pychunkedgraph/meshing/meshgen.py

```diff
@@ -360,6 +360,48 @@ def chunk_mesh_task(cg, chunk_id, cv_path,
                      len(manifests_to_upload) - len(fragments_to_upload)))
 
 
+def mesh_lvl2_previews(cg, lvl2_node_ids, cv_path=None,
+                       cv_mesh_dir=None, mip=2, simplification_factor=999999,
+                       max_err=40, parallel_download=8, verbose=True,
+                       cache_control=None, n_threads=4):
```

Comment:
Since you eventually want to overwrite the preview with a real mesh, it would be better to set `cache_control` to `'no-cache'`, otherwise users will have to wait for an hour until the real mesh shows up.